doc/reference/configuration/configuration_reference.rst

```diff
@@ -2389,3 +2389,345 @@ The ``security`` section defines configuration parameters related to various sec
+    to store them on different physical disks for performance matters.
+
+    |
+    | Type: string
+    | Default: 'var/lib/{{ instance_name }}'
+    | Environment variable: TT_WAL_DIR
+
+..  _configuration_reference_wal_dir_rescan_delay:
+
+..  confval:: wal.dir_rescan_delay
+
+    Number of seconds between periodic scans of the write-ahead-log
+    file directory, when checking for changes to write-ahead-log
+    files for the sake of :ref:`replication <replication>` or :ref:`hot standby <index-hot_standby>`.
+
+    |
+    | Type: number
+    | Default: 2
+    | Environment variable: TT_WAL_DIR_RESCAN_DELAY
+
+..  _configuration_reference_wal_max_size:
+
+..  confval:: wal.max_size
+
+    The maximum number of bytes in a single write-ahead log file.
+    When a request would cause an ``.xlog`` file to become larger than
+    ``wal.max_size``, Tarantool creates a new WAL file.
+
+    |
+    | Type: integer
+    | Default: 268435456
+    | Environment variable: TT_WAL_MAX_SIZE
+
+..  _configuration_reference_wal_mode:
+
+..  confval:: wal.mode
+
+    Specify fiber-WAL-disk synchronization mode as:
+
+    *   ``none``: write-ahead log is not maintained.
+        A node with ``wal.mode = none`` can't be replication master.
+
+    *   ``write``: :ref:`fibers <fiber-fibers>` wait for their data to be written to
+        the write-ahead log (no ``fsync(2)``).
+
+    *   ``fsync``: fibers wait for their data, :manpage:`fsync(2)`
+        follows each :manpage:`write(2)`.
+
+    |
+    | Type: string
+    | Default: 'write'
+    | Environment variable: TT_WAL_MODE
+
+..  _configuration_reference_wal_queue_max_size:
+
+..  confval:: wal.queue_max_size
+
+    The size of the queue in bytes used by a :ref:`replica <replication-roles>` to submit
+    new transactions to a :ref:`write-ahead log <internals-wal>` (WAL).
+    This option helps limit the rate at which a replica submits transactions to the WAL.
+
+    Limiting the queue size might be useful when a replica is trying to sync with a master and
+    reads new transactions faster than writing them to the WAL.
+
+    ..  NOTE::
+
+        You might consider increasing the ``wal.queue_max_size`` value in case of
+        large tuples (approximately one megabyte or larger).
+
+    |
+    | Type: integer
+    | Default: 16777216
+    | Environment variable: TT_WAL_QUEUE_MAX_SIZE
+
+..  _configuration_reference_wal_ext:
+
+wal.ext.*
+~~~~~~~~~
+
+..  admonition:: Enterprise Edition
+    :class: fact
+
+    Configuring ``wal.ext.*`` parameters is available in the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+This section describes options related to :ref:`WAL extensions <wal_extensions>`.
+The options allow you to add auxiliary information to each :ref:`write-ahead log <internals-wal>` record.
+
+Note that records with additional fields are :ref:`replicated <replication-architecture>` as follows:
```

Comment:
Let's remove these details because they already documented in the topic. I'd keep introductions for reference options short to avoid duplication with main topics.

Reply:
Removed this part. Thanks!